openstf/model/trainer/xgboost/xgboost.py

```diff
@@ -176,35 +197,27 @@ def better_than_old_model(self, test_data, penalty_factor=1.2):
                 )
             except Exception as e:
                 prediction_old_model = np.nan
```

Comment:
This variable seems not to be used further in the code

Reply:
thanks, I'll remove it!